Source/Core/Core/BootManager.cpp

```diff
@@ -433,11 +433,23 @@ bool BootCore(std::unique_ptr<BootParameters> boot, const WindowSystemInfo& wsi)
   if (StartUp.bWii && DiscIO::IsNTSC(StartUp.m_region) && Config::Get(Config::SYSCONF_PAL60))
     Config::SetCurrent(Config::SYSCONF_PAL60, false);
 
-  // Ensure any new settings are written to the SYSCONF
   if (StartUp.bWii)
   {
-    Core::BackupWiiSettings();
-    ConfigLoaders::SaveToSYSCONF(Config::LayerType::Meta);
+    const bool want_determinism = Movie::IsMovieActive() || NetPlay::IsNetPlayRunning();
```

Comment:
Is it possible to use Core::UpdateWantDeterminism(true) and Core::WantsDeterminism here?

edit: If we use UpdateWantDeterminism, the call would have to be outside the bWii conditional.

Reply:
I've added a commit that does this.